htdocs/landing/aup.html

```diff
@@ -0,0 +1,104 @@
+<!doctype html>
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <title>GOCDB Privacy Notice</title>
+  <link rel="stylesheet" type="text/css" href="home.css" />
+  <link rel="SHORTCUT ICON" href="Logo-1.4-FavIcon-32x32.regional.ico" />
+</head>
+
+<div class="wide_page_container" >
+  <div class="centre_box">
+    <div class="centrePageContainer">
+      <div class="policyText">
+        <!--Start of policy text output by PANDOC -->
+        <h1 id="acceptable-use-policy-and-conditions-of-use">Acceptable Use Policy and Conditions of Use</h1>
+        <p>Your use of the Grid Operations Centre Database is governed by the following Acceptable Use Policy and Conditions of Use.</p>
+        <p>This policy is effective from 10/10/2016 and replaces an earlier version of this document [R1]. This policy is one of a set of documents that together define the Security Policy [R2]. This individual document must be considered in conjunction with all the policy documents in the set.</p>
```

Comment:
Use in page links for the references like the Privacy Notice does?

Reply:
Response as above. The text is converted from the EGI policy. (I might not have understood the comment)

Reply:
Ah, so there isn't a nice way to add in-page links as this is auto-generated?

Reply:
Not really no, not a "nice" way.